testsuite/features/secondary/srv_content_lifecycle.feature

```diff
@@ -157,3 +157,14 @@ Feature: Content lifecycle
     And I click on "Delete"
     And I click on "Delete" in "Delete Project" modal
     Then I should not see a "clp_name" text
+    When I delete these channels with spacewalk-remove-channel:
```

Comment:
Split the scenario in two scenarios, with names following our conventions:

* Cleanup: remove the Content Lifecycle Management project
* Cleanup: remove the created channels

Reply:
Suggestion: add a verification like:
 
  Then there should be no channel starting with "clp"

? feel free to ignore

Reply:
I would agree with Eric's first suggestion.

Reply:
I agree with both suggestions. But please using "clp_label" for the regex of the second suggestion.